### PR TITLE
Tech-debt: correct types for house mnmgnt and refactoring

### PR DESCRIPTION
**Pull request by Maria358**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by rostovskiy88 — DISMISSED** [review]

File: src/houses/components/ManageHouseForm.vue [hunk 1]
```diff
@@ -181,26 +181,16 @@ import useVuelidate from '@vuelidate/core';
...
-    cooperationId: {
-      type: Number,
-      required: true,
-    },
-    houseData: {
-      type: HouseModel,
-      required: true,
-    },
-  },
+  props: ['cooperationId', 'houseData'],
```
Comment (tsehla42) [inline src/houses/components/ManageHouseForm.vue]: Using props as array conaidered as a bad practice

File: src/components/CreateInvitationForm.vue [hunk 2]
```diff
@@ -99,13 +99,12 @@
 import { defineComponent } from 'vue';
 
 import Button from 'primevue/button';
-import Dialog from 'primevue/dialog';
 import InputText from 'primevue/inputtext';
 import Dropdown from 'primevue/dropdown';
 import useVuelidate from '@vuelidate/core';
 import { StoreModuleEnum } from '@/store/types';
 import { mapGetters } from 'vuex';
```
Comment (tsehla42) [inline src/components/CreateInvitationForm.vue]: Wrong import order

File: src/houses/components/ManageHouseForm.vue (hunk 1 above)
Comment (Maria358) [reply src/houses/components/ManageHouseForm.vue]: Agree

File: src/components/CreateInvitationForm.vue [hunk 3]
```diff
@@ -99,13 +99,12 @@
...
-import { ApartmentsActionsEnum, ApartmentsGettersEnum } from '@/apartment/store/apartments/types';
+import { ApartmentInterface, ApartmentsActionsEnum, ApartmentsGettersEnum } from '@/apartment/store/apartments/types';
 import {
   requiredValidator,
   emailValidator,
```
Comment (tsehla42) [inline src/components/CreateInvitationForm.vue:109]:
```suggestion
import { defineComponent } from 'vue';
import { mapGetters } from 'vuex';
import InputText from 'primevue/inputtext';
import Dropdown from 'primevue/dropdown';
import useVuelidate from '@vuelidate/core';
import { StoreModuleEnum } from '@/store/types';
import { ApartmentInterface, ApartmentsActionsEnum, ApartmentsGettersEnum } from '@/apartment/store/apartments/types';
import {
  requiredValidator,
  emailValidator,
	@@ -114,7 +113,7 @@ import {
  emailLastCharsValidator,
} from '@/utils/validators';
import { InvitationsActionsEnum, InvitationTypesEnum, PostInvitationInterface } from '@/store/invitations/types';
import { HousesActionsEnum, HousesGettersEnum, HouseInterface } from '@/houses/store/types';
import { CooperationGettersEnum } from '@/cooperation/store/types';
```

Comment (Maria358) [reply src/components/CreateInvitationForm.vue:109]: Thanks!

**Review by rostovskiy88 — APPROVED** [review]

**Review by tsehla42 — APPROVED** [review]

**Pull request merged**